library/core/src/primitive_docs.rs

```diff
@@ -801,11 +801,57 @@ mod prim_array {}
 /// assert_eq!(2 * pointer_size, std::mem::size_of::<Box<[u8]>>());
 /// assert_eq!(2 * pointer_size, std::mem::size_of::<Rc<[u8]>>());
 /// ```
+///
+/// ## Trait Implementations
+///
+/// There are several implementations of common traits for slices. Some examples
+/// include:
+///
+/// * [`Clone`]
+/// * [`Eq`], [`Ord`] - for slices whose element type are [`Eq`] or [`Ord`].
+/// * [`Hash`] - for slices whose element type is [`Hash`].
```

Comment:
It feels a little odd that Eq/Ord are combined into one item here but Hash is separate. I think it makes sense to keep all lines separate or all together; I'd personally prefer separate.

Reply:
This section could just be a couple sentences:

```rust
/// Some traits are implemented for slices if the element type implements
/// that trait. This includes [`Eq`], [`Hash`] and [`Ord`].
```